x-pack/test/cloud_security_posture_functional/helpers.ts

```diff
@@ -4,24 +4,38 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
 
-const CLOUD_SECURITY_POSTURE_PACKAGE_NAME = 'cloud_security_posture';
+const CLOUD_SECURITY_PACKAGE_NAME = 'cloud_security_posture';
+const CLOUD_SECURITY_PACKAGE_VERSION = '1.0.8';
 
 /**
- * flags to load kibana with fleet pre-configured to have 'cloud_security_posture' integration installed
+ * Package version is fixed (not latest) so FTR won't suddenly break when package is changed.
+ *
+ * test a new package:
+ * 1. merge a new package to EPR
+ * 2. locally checkout the kibana version that matches the new package
+ * 3. update CLOUD_SECURITY_PACKAGE_VERSION with the new package version
+ * 4. run FTR test locally
+ * 5. when test pass, merge new kibana changes with the new package version.
+ *
```

Comment:
the workflow described here is how we should be developing packages for EPR while making sure:
 - we don't suddenly break FTR on CI
 - we test the new package with its matching kibana version and FTR tests.
 
 worth noting that this is a work in progress and hopefully we'll have a better way of achieving the same results in the future.

Reply:
uh, i think there's an issue with this workflow. looking into it. 

say next release is `8.7` and we need to push a new package for `8.6`. we'd first publish the new package, which would reach users and only after it's in EPR we could pull it and test in kibana and then backport the changes to `8.6`, which means we could introduce errors in production until the backport is complete. this isn't a problem with `8.7` as it's not released yet.

Reply:
alright, scratch that. wasn't good anyway 😄 

to release a new package, for any version, first start a package registry locally with `NODE_EXTRA_CA_CERTS` and ru n the tests with your new package.